pathfinding_service.rst

```diff
@@ -341,18 +177,11 @@ Example
         "errors": "No suitable path found."
     }
 
-
-Implementation notes
-====================
-
-Network topology updates
+Network Topology Updates
 ------------------------
 
 .. Note::
-    A pathfinding service might want to cover multiple token networks. However,
-    it always needs to cover the *RDN* token network in order to be able to
-    provide routing information for payments.
-    
+    A pathfinding service might want to cover multiple token networks.
```

Comment:
When do we have two separate PFSs? When they run in two processes? When they have to different ethereum addresses for payment? When they use two different IP addresses? So I'm not sure what this statement actually means.

Reply:
I guess my feedback was not very actionable in this case. I just wanted to state that the meaning of that sentence is unclear to me. We can either remove it or add more details to it.

Depending on our decisions regarding the payment, if might be helpful if a single ethereum address could be used to pay for routes in multiple token networks, since that could reduce the required number of payment channels if we go with a payment channel solution.

It's probably best to remove it until we know what exactly we want to say here, which I don't at this point.

Reply:
Thanks for the clarification @karlb !